models/course.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models, fields
+
+
+class Course(models.Model):
```

Comment:
Class name same as model please.

```suggestion
class OpenAcademyCourse(models.Model):
```